db/migrate/20210718074605_update_ibm_vendor_to_ibm_cloud.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateIbmVendorToIbmCloud < ActiveRecord::Migration[6.0]
+  class Vm < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time('Updating vendor to "IBM Cloud" for VPC managers') do
+      Vm.in_my_region.where(:type => 'ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm').update_all(:vendor => 'ibm_cloud')
```

Comment:
I think I also need to update the template (image) records, but `VPC::CloudManager` and `PowerVirtualServers::CloudManager` don't subclass `VmOrTemplate` (https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/cloud_manager/vm_or_template.rb#L1). Should I update `Template` records separately?

Reply:
I have honestly never seen that particular VmOrTemplate class before and it's blowing my mind 🤯 

Even so, in migrations we ignore the STI type (via line 4), so you will need to add both types to the :type clause in the query.
```suggestion
      Vm.in_my_region.where(:type => ['ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm', 'ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm']).update_all(:vendor => 'ibm_cloud')
```

Reply:
> Even so, in migrations we ignore the STI type (via line 4), so you will need to add both types to the :type clause in the query.

I found an existing migration and tried to do something similar - will this work?
https://github.com/ManageIQ/manageiq-schema/blob/master/db/migrate/20200607025146_add_ancestry_to_vm.rb#L2

Reply:
What you have here https://github.com/ManageIQ/manageiq-schema/pull/587/files#diff-36fa40d8b3ea49df31e7c2b95ed2a9b80bd6fc326a6d673850146a45693925cbR2-R6 looks good